CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   (#[1547](https://github.com/nix-rust/nix/pull/1547))
 - Added getter methods to `MqAttr` struct
   (#[1619](https://github.com/nix-rust/nix/pull/1619))
+- Added `timer` support
```

Comment:
"timer" is a bit vague.  "timer_create" would be more descriptive, or even "POSIX per-process timers".